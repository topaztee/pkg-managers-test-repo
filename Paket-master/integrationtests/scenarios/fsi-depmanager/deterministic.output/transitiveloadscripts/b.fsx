#load "skip.c.fsx"
#r "paket: nuget FSharp.Data"
printfn "---->b.fsx"
open FSharp.Data
open NUnit
printfn "---->b.fsx///"