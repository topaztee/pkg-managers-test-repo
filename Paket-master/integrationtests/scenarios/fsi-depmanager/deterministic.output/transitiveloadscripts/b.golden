---->c.fsx
---->b.fsx
---->b.fsx///
